.github/workflows/comment-npm-publish.yml

```diff
@@ -41,9 +41,24 @@ jobs:
           git checkout pr-find-commit
 
       - uses: actions/setup-node@v3
+      - name: Check for .nvmrc file root folder
+        continue-on-error: true
```

Comment:
Continue on error = true ?

Reply:
Don't ask me, I just copy and paste ;) But I think it means it shouldn't abandon, but carry out the next step which is the fallback to default (14).

Reply:
Yea, because we use the exit code in an if statement, not to fail the build.